allennlp/pretrained.py

```diff
@@ -25,7 +25,13 @@ def predictor(self) -> Predictor:
 
```

Comment:
This file is actually moving to https://github.com/allenai/allennlp-hub/blob/master/allennlp_hub/pretrained/allennlp_pretrained.py and allennlp_semparse_pretrained.py. You should might be able to generate a `diff` and `patch` `allennlp_pretrained.py` in the hub repo. Sorry for the trouble :(

Reply:
Not a big deal--the hard part was fighting with the doc formatting...

Shouldn't I just leave it in here for now though and merge forward?  I can apply a separate patch on `allennlp_hub`.  Alternatively I can revert my changes to `pretrained.py` with `git checkout master -- allennlp/pretrained.py`.

Reply:
NM--I see I have a conflict...

Reply:
Yep, that's my fault. Sorry about that. `allennlp-hub` is now the sole source for pretrained artifacts so this shouldn't be an issue going forward.